### ISSUE TITLE
http admin cannot connect to server

### ISSUE THREAD

**Issue by osckal** [rethinkdb/rethinkdb#6170]
OS: windows 10 pro (latest update)

any browser (edge, chrome, firefox), regardlass of http port, i will always get "Connected to <unknown>" in the http admin page.  i tried to run a server instance from a win7pro VM and the win10. i can connect to both win10 and win7 http admin from the VM. from win10, i still get the same behavior when i try to access to http admin of the win7 vm. it looks like its isolated to win10

from windows7 pro VM (192.168.1.248 is win10 server instance)
![rdb-win7](https://cloud.githubusercontent.com/assets/21185820/20026824/95dad442-a33d-11e6-8577-928d44d16191.PNG)

from windows10 (192.168.1.248 is win10 server instance)
![rdb-win10](https://cloud.githubusercontent.com/assets/21185820/20026825/95db69fc-a33d-11e6-9215-87cf3930d7d1.PNG)

**Comment by w4567892015** [comment]
Yes it's Bug, My computer Win10. I use localhost UI admin connect it, But is same as you. The other computer use this is normal.

**Comment by AtnNn** [comment]
@oskal @w4567892015 Are you able to connect to your RethinkDB instance using a different browser?

The RethinkDB web UI does not yet work in the Edge browser. See https://github.com/rethinkdb/rethinkdb/issues/5878

**Comment by w4567892015** [comment]
I used browser is firefox or google not ie
